quickstarts/fullstack/next.mdx

```diff
@@ -104,30 +105,27 @@ export default function LoginPage() {
 
 <Tab title="Pages directory">
 ```jsx pages/login.tsx
-import dynamic from "next/dynamic";
+import dynamic from 'next/dynamic';
+const HankoAuth = dynamic(() => import('@/components/HankoAuth'), { ssr: false })
 
 export default function LoginPage() {
-  const HankoAuth = dynamic(
-    () => import('@/components/HankoAuth'),
-    { ssr: false },
-  )
   return (
-    <Suspense fallback={"Loading ..."}>
       <HankoAuth/>
-    </Suspense>
-  )
+  );
 }
 ```
+</Tab>
+</Tabs>
+
 <Info>
-**Why we use dynamic import in Nextjs pages directory?** 🤔
+**Why we use dynamic import in Nextjs?** 🤔
```

Comment:
Up to you but maybe add that this is because we use customEvents and especially the WebAuthn API for passkeys and both are only available on the client.

Reply:
Okay will add this. It would be good to mention